src/main/java/frc/trigon/robot/subsystems/collector/simulationcollector/SimulationCollectorIO.java

```diff
@@ -0,0 +1,25 @@
+package frc.trigon.robot.subsystems.collector.simulationcollector;
+
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.trigon.robot.subsystems.collector.CollectorIO;
+import frc.trigon.robot.subsystems.collector.CollectorInputsAutoLogged;
+
+public class SimulationCollectorIO extends CollectorIO {
+    private final DCMotorSim dcMotorSim = SimulationCollectorIOConstants.DC_MOTOR_SIM;
+
+    @Override
+    protected void updateInputs(CollectorInputsAutoLogged inputs) {
+        inputs.motorVoltage = dcMotorSim.getOutput().get(0, 0);
```

Comment:
have a local variable updated from the `setPower` command here to get the motor voltage